### ISSUE TITLE
[BUG] Persistent Markers not appearing for JIP admins

### ISSUE THREAD

**Issue by AUTigerGrad** [ALiVEOS/ALiVE.OS#271]
When I rejoin my server after a player exit, all persistent alive markers that I left on my last session do not appear to be when logged in as admin.

When I completely relaunch the server after a server save and exit, all markers reappear.

**Comment by tupolov** [comment]
with latest 1.2.8 code? We tested heavily (after heavy drinking) at the hackathon.

**Comment by AUTigerGrad** [comment]
yup with the latest code.  It's "hit or miss"..some folks will see the markers after JIP...some won't.

**Comment by Vegas** [comment]
@tupolov Also experiencing this same issue. This has nothing to do with who is admin and who isn't.

The markers simply do not show up for all players after JIP. It works about 60% of the time.

**Comment by highhead** [comment]
can confirm this

**Comment by Repolak** [comment]
Agreed, this is happening with any JIP player, not just admins. Upon everyone joining and launching the mission from the start, no one has issues.

**Comment by highhead** [comment]
rejoining da mish fixes it, but its causing unneeded traffic
